test: admin can kick a member and delete message

Valentina1133 opened the pull request:
Community admin can delete member's message and kick a member test automated
CI run (all tests): 
https://ci.status.im/job/status-desktop/job/e2e/job/manual/1748/allure/ 

<img width="1260" alt="Screenshot 2024-04-08 at 15 07 18" src="https://github.com/status-im/desktop-qa-automation/assets/141633821/5af5b1db-a5f2-4596-899d-4e0584577b51">

Valentina1133 commented:
@anastasiyaig I placed this test in test_communities.py file, but now I think maybe it's better to create a separate file? 
Also - we can add steps from this scenario to test_join_community_and_pin_unpin_message - it reduces time of tests but the scenario will be much longer. What do you think?

anastasiyaig commented:
file is fine i think! Lets keep it as is for now
pls resolve 1 conflict . Btw regarding it, i dont know how test is passing having wrong string for permission deletion (i fixed the string itself but before i changed that - tests were not failing :D 

https://github.com/status-im/desktop-qa-automation/blob/74dfc258c7ddeba44845c96ec8a9056c5d4630d4/constants/community_settings.py#L35

we might want to check on the verification for the toasts in permission tests (separate task)

Valentina1133 commented:
> file is fine i think! Lets keep it as is for now pls resolve 1 conflict . Btw regarding it, i dont know how test is passing having wrong string for permission deletion (i fixed the string itself but before i changed that - tests were not failing :D
> 
> https://github.com/status-im/desktop-qa-automation/blob/74dfc258c7ddeba44845c96ec8a9056c5d4630d4/constants/community_settings.py#L35
> 
> we might want to check on the verification for the toasts in permission tests (separate task)

Yes! You are right!

I think I know why. Because in permissions test we check that toast message from constants is in list of toast messages appeared in test, because there are several toast messages at the same time. So I believe we need to check it element by element using indexes to check every toast message separately.
